src/Microsoft.AspNetCore.Razor.Tools/Client.cs

```diff
@@ -12,6 +12,23 @@ namespace Microsoft.AspNetCore.Razor.Tools
 {
     internal abstract class Client : IDisposable
     {
+        private static int counter;
```

Comment:
lol super hack. Why isnt a guid (or even getting the current instances hash code) sufficient?

Reply:
We do the same thing here as well https://github.com/aspnet/Razor/blob/dev/src/Microsoft.AspNetCore.Razor.Tools/ConnectionHost.cs#L20. This is only used for logging purposes. I think the reason for the counter was to log the order in which the clients get connected/disconnected.

Reply:
Fair enough 👍